src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/InsertAnnotationMissingQuickFix.java

```diff
@@ -73,6 +80,25 @@ public List<? extends CodeAction> getCodeActions(JavaCodeActionContext context,
         return codeActions;
     }
 
+    @Override
+    public CodeAction resolveCodeAction(JavaCodeActionResolveContext context) {
+        final CodeAction toResolve = context.getUnresolved();
+        String name = getLabel(annotations);
```

Comment:
Should retrieve the title from the code action instead of recomputing it here.

Reply:
Made the necessary changes so that the title is retrieved from `toResolve.getTitle()` .